app/client.js

```diff
@@ -16,16 +16,51 @@ class Client {
   }
 
   /**
-   * @param {HttpMethod} method
+   * @typedef {Object} Filters
```

Comment:
Please add the `Filters` object to the `index.d.js`.

Reply:
> Please add the `Filters` object to the `index.d.js`.

Added.